dataprocspawner/customize_cluster.py

```diff
@@ -57,240 +145,787 @@ def get_custom_cluster_html_form(autoscaling_policies, node_types):
+            <span class="mdc-list-item__text">SSD Persistent Disk</span>
+          </li>
+        </ul>
+      </div>
+    </div>
     <br />
-    <div class="form-group" name="sec_worker">
-      <label for="sec_worker"><h1>Secondary worker nodes</h1></label>
-      <div class="form-group">
-        <label for="sec_worker_disk_type">Primary disk type</label>
-        <select class="form-control" name="sec_worker_disk_type">
-          <option value="pd-standard">Standard Persistent Disk</option>
-          <option value="pd-ssd">SSD Persistent Disk</option>
-        </select>
-      </div>"""
+    <br />"""
 
   html_secondary_worker_disk_size = """
-    <div class="form-group">
-      <label for="sec_worker_disk_size">Primary disk size</label>
-      <input name="sec_worker_disk_size" class="form-control"
-        placeholder="default" value=''></input>
-    </div>"""
+    <label class="mdc-text-field jupyter-text-field mdc-text-field--outlined">
+      <span class="mdc-notched-outline">
+        <span class="mdc-notched-outline__leading"></span>
+        <span class="mdc-notched-outline__notch">
+          <span class="mdc-floating-label" id="sec_worker_disk_size">
+            Primary disk size
+          </span>
+        </span>
+        <span class="mdc-notched-outline__trailing"></span>
+      </span>
+      <input
+        type="text"
+        class="mdc-text-field__input"
+        name="sec_worker_disk_size"
+        aria-labelledby="sec_worker_disk_size"
+        value="500"/>
+    </label>
+    <br />
+    <br />"""
 
   html_secondary_worker_amount = """
-    <div class="form-group">
-      <label for="sec_worker_node_amount">Number of secondary worker nodes</label>
-      <input name="sec_worker_node_amount" class="form-control" placeholder="" value='0'></input>
-    </div>
-    </div>"""
+    <label class="mdc-text-field jupyter-text-field mdc-text-field--outlined">
+      <span class="mdc-notched-outline">
+        <span class="mdc-notched-outline__leading"></span>
+        <span class="mdc-notched-outline__notch">
+          <span class="mdc-floating-label" id="sec_worker_node_amount">
+            Number of secondary worker nodes
+          </span>
+        </span>
+        <span class="mdc-notched-outline__trailing"></span>
+      </span>
+      <input
+        type="text"
+        class="mdc-text-field__input"
+        name="sec_worker_node_amount"
+        aria-labelledby="sec_worker_node_amount"
+        value="0"/>
+    </label>
+    <br />
+    <br />"""
+
+  html_adv_base = """
+    <h2 class="jupyter-form__group-title">Advanced</h2>
+    <br />"""
 
   html_custom_labels = """
-    <br />
-    <div class="form-group">
-      <label for="custom_labels">User defined labels</label>
-      <input name="custom_labels" class="form-control" placeholder="key1:value1,key2:value2"
-        value=''>
-      </input>
-    </div>"""
+    <label class="mdc-text-field jupyter-text-field mdc-text-field--outlined">
```

Comment:
input label missing here https://screenshot.googleplex.com/3sbBrNmyqFJN99j

Reply:
It's not an issue, that's how outlined material text field works.
You can see demo [here](https://material-components.github.io/material-components-web-catalog/#/component/text-field)